src/modules/mc_att_control/mc_att_control_params.c

```diff
@@ -335,11 +339,9 @@ PARAM_DEFINE_FLOAT(MC_PITCHRATE_MAX, 220.0f);
 /**
  * Max yaw rate
  *
- * A value of significantly over 120 degrees per second can already lead to mixer saturation.
- *
  * @unit deg/s
  * @min 0.0
- * @max 360.0
+ * @max 3600.0
```

Comment:
Should be 1800, as 2000 is the hardware limit. We should bound it to what we can still sense.